modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -432,3 +432,83 @@ Status description:: error: syntax error or access rule violation - auth info ex
 === 42NFF
 
 Status description:: error: syntax error or access rule violation - permission/access denied. Access denied, see the security logs for details.
+
+== Procedure exception
+
+Procedure exceptions occur when there is an error in executing a procedure, such as when the procedure execution fails due to a client error, when the procedure cannot be invoked on a primary, when the number of arguments to checkConnectivity is invalid, etc.
```

Comment:
I felt like some examples here are maybe a bit too specific, but also we must maybe do the investigation task about external procedure that we have in the Q4 doc to be able to explain this better, so I think it can work for now